utils/middleware.ts

```diff
@@ -0,0 +1,19 @@
+import { param, validationResult } from "express-validator";
+import { Request, Response, NextFunction } from "express";
+import { ALLOWED_NETWORKS } from ".";
+
+/**
+ * Validations for the params of the challenge submission route
+ */
+export const validateSubmission = [
+  param("challengeId").isInt().withMessage("Challenge ID must be an integer"),
```

Comment:
I have never used express-validator. This looks really convenient!

Reply:
yeah its more convenient that writing the logic from scratch and helps with readability. So similar reasoning for using it as `axios`